lib/color.dart

```diff
@@ -25,32 +24,42 @@ class Color {
   /// For color with specified [alpha] component output format will be rgba.
   const Color([this.red = 0, this.green = 0, this.blue = 0, this.alpha]);
 
-  bool operator ==(Color other) {
-    return red == other.red && green == other.green && blue == other.blue &&
-        alpha == other.alpha;
+  bool operator ==(Object other) {
+    if (other is Color) {
```

Comment:
What was it done for? Do you need compare objects of the Colors with any objects?

Reply:
what about other operations?